agent/agents/mysql/perfschema/perfschema.go

```diff
@@ -258,7 +260,17 @@ func (m *PerfSchema) Run(ctx context.Context) {
 }
 
 func (m *PerfSchema) runHistoryCacheRefresher(ctx context.Context) {
-	t := time.NewTicker(refreshHistory)
+	interval := refreshHistory
+	if value := os.Getenv("PMM_PERFSCHEMA_REFRESH_RATE"); value != "" {
```

Comment:
it's interesting, we never before had any env variable set directly to PMM Client except PMM_DEBUG and PMM_TRACE. What do you think if adding additional field to pmm-agent.yml would work better?

Reply:
Up to you. So you want to have property in config and also env variable?

Reply:
let's have only config for now